assign_rights/tests.py

```diff
@@ -226,6 +226,11 @@ def test_create_json(self):
             start_date = random_date(75, 50).isoformat()
             end_date = random_date(49, 5).isoformat()
             serialized = self.assembler.create_json(obj, serializer_cls, start_date, end_date)
+            if obj_cls == RightsShell:
+                if obj.jurisdiction:
+                    self.assertEqual(serialized['jurisdiction'], 'us')
+                else:
+                    self.assertEqual(serialized['jurisdiction'], None)
```

Comment:
I'd just assert that the string is lowercase if it exists. The second test clause isn't really testing anything ("if there's nothing there, assert there's nothing there"), and checking against a specific string is probably a little too restrictive here.
```suggestion
                if obj.jurisdiction:
                    self.assertTrue(serialized['jurisdiction'].islower())
```